### PR TITLE
Init codeowners

### PR DESCRIPTION
**Pull request by mojito317**
## PR summary

The reason for this PR to restrict pull requests against files that are generated.

## PR Checklist

Please make sure that your PR fulfills the following requirements:

- [x] The commit message follows the
[Angular Commit Message Guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines).
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type  

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] New tests
- [x] Build/CI related changes
- [x] Documentation content changes
- [ ] Other (please describe)

## What is the current behavior?
PRs can be merged for generated files without the approval of sdks team members.

## What is the new behavior?

The listed files in [CODEOWNERS](.github/CODEOWNERS) can no longer be pushed before the @IBM/cloudant-sdks members have not reviewed it.
This also restricts the PRs from @cloudant-sdks-automation: the @IBM/cloudant-sdks must review this PR before the merge.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by bessbd — APPROVED** [review]

**Review by ricellis — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS
Comment (mojito317) [reply .github/CODEOWNERS]: I added it in https://github.com/IBM/cloudant-python-sdk/pull/29/commits/2bbf866585acb2f4223413c775afa3884a966ad2.

File: CONTRIBUTING.md
Comment (mojito317) [reply CONTRIBUTING.md]: I reverted it in 8ed263f.

File: CONTRIBUTING.md
Comment (mojito317) [reply CONTRIBUTING.md]: Thanks, I was not sure about my sentences as well, this is a much nicer served paragraph. Fixed in 9ec1982.

**Review by ricellis — APPROVED** [review]
+1

**Comment by mojito317** [comment]
@bessbd and @ricellis thank you for the reviews! I'm going to squash and merge this PR ASAP.

File: CONTRIBUTING.md
Comment (emlaver) [inline CONTRIBUTING.md:20]: We should remove `partly` as it opens the door for questions like "which parts?"
I've also suggested some other changes:
```suggestion
This repository is generated from the Cloudant OpenAPI
specification. PRs to source code, tests and markdown documentation
might be improper if the change is required within the OpenAPI specification.
Please [review the list of files](.github/CODEOWNERS) that _should not_ be modified.
```

File: .github/CODEOWNERS
Comment (emlaver) [inline .github/CODEOWNERS:6]: We might want to select the whole folder:
```suggestion
.github/** @IBM/cloudant-sdks
scripts/** @IBM/cloudant-sdks
stubs/** @IBM/cloudant-sdks
```

**Review by emlaver — COMMENTED** [review]
Some thoughts

File: CONTRIBUTING.md
Comment (mojito317) [reply CONTRIBUTING.md:20]: For me, it is clear the `partly` refers to the list of files mentioned in one of the later sentences. That is why I would keep the first sentence as-is, mainly because not the whole repository, but a part of it is generated. However, I fancy the simplification of the second sentence.

What do you think about this:
```suggestion
The code in the repository is partly generated from the Cloudant OpenAPI
specification. PRs to source code, tests, and markdown
might be improper if the change is required within the OpenAPI specification. 
Please, first [look at the list of files](.github/CODEOWNERS) that
_should not_ be modified.
```

File: .github/CODEOWNERS
Comment (mojito317) [reply .github/CODEOWNERS:6]: I think it is not necessary, the scope of this task to add the files that are generated somehow. `.github` has already some language-specific files in it. `scripts`, `stubs` might have as well. But we can turn back to this any time when we feel it is necessary.

File: CONTRIBUTING.md
Comment (mojito317) [reply CONTRIBUTING.md:12]: Since the sentence is starting in plural first person and tries to be informal here, grammatically I do not see this modification as reasonable.

Maybe if the whole sentence would be formal, but I like the informal version better:
```suggestion
The Cloudant team asks that an **issue is always opened prior to a PR** to give them the
```

File: .github/CODEOWNERS
Comment (vmatyusGitHub) [reply .github/CODEOWNERS:6]: I think it is necessary. What if a new file is added to these directories? Or a Whole new directory can be added to these folders and we do not get notification about it.

File: CONTRIBUTING.md
Comment (vmatyusGitHub) [inline CONTRIBUTING.md:20]: `Please, first [look at the list of files](.github/CODEOWNERS) that
_should not_ be modified.`

I think this statement invalid.

The codeowner setting does not deny the modification of files, but notifies the team about a PR is opened and requires a review before merge.

> Code owners are automatically requested for review when someone opens a pull request that modifies code that they own.
Source: https://docs.github.com/en/free-pro-team@latest/github/creating-cloning-and-archiving-repositories/about-code-owners

File: .github/CODEOWNERS
Comment (vmatyusGitHub) [inline .github/CODEOWNERS:1]: I think it is better to set a default owner:
```
* @IBM/cloudant-sdks
```

File: CONTRIBUTING.md
Comment (vmatyusGitHub) [reply CONTRIBUTING.md:20]: I think we should collect rather which files can be modified by contributors and in which area we expect extensions.

File: .github/CODEOWNERS
Comment (emlaver) [reply .github/CODEOWNERS:6]: I'm happy with keeping this as-is after our Slack call.

File: CONTRIBUTING.md
Comment (emlaver) [reply CONTRIBUTING.md:12]: No need to change, let's keep as-is.

File: CONTRIBUTING.md
Comment (emlaver) [reply CONTRIBUTING.md:20]: No need to change, keep as-is.

**Review by emlaver — APPROVED** [review]

**Review by vmatyusGitHub — APPROVED** [review]

File: .github/CODEOWNERS
Comment (vmatyusGitHub) [reply .github/CODEOWNERS:6]: Yeah, we can keep it as-is.

**Comment by mojito317** [comment]
Thanks for the approvals @vmatyusGitHub and @emlaver! I'm about to merge this.

**Pull request merged**
